src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -189,29 +189,56 @@ private function replaceByShortType(Tokens $tokens, TypeAnalysis $type, array $u
      */
     private function getTypes(Tokens $tokens, int $index, int $endIndex): iterable
     {
-        $index = $typeStartIndex = $typeEndIndex = $tokens->getNextMeaningfulToken($index - 1);
-        $type = $tokens[$index]->getContent();
-
+        $skipNextYield = false;
+        $typeStartIndex = $typeEndIndex = null;
+        $type = null;
```

Comment:
```suggestion
        $type = $typeStartIndex = $typeEndIndex = null;
```
It's like this in few other places, I don't want to spam PR with similar comments, so please change this in other places too if you think that suggestion is applicable.

Reply:
These variables are semantically not the same. Indexes are of type `int|null` and `$type` of type `string|null`. Thus marking as resolved, but I do not want merge this change as it might to the same thing, but it is illogical.

Reply:
Yeah, I get it, but in terms of assigning `null` it does not make any difference if these variables may contain different type later. It would make sense to distinguish it if we had local types or even `@var`. But I won't fight for it 😉.